src/entities/Category/types.ts

```diff
@@ -17,3 +17,5 @@ export enum DecentralandCategories {
 
 // TODO: review this type with the other ones: naming and maybe we can merge some (@lauti7)
 export type Category = { name: string; active: boolean; count?: number }
+
+export type CategoryCountTarget = "all" | "places" | "worlds"
```

Comment:
What do you think about changing this to a enum

If you need this to be a type, maybe something like 

```suggestion
export enum CategoryCountTargetOptions {
  ALL = "all",
  PLACES = "places",
  WORLDS = "worlds",
}
export type CategoryCountTarget = CategoryCountTargetOptions
```